### PR TITLE
Register-user

### PR DESCRIPTION
**Pull request by judytabaca**

### CODE REVIEW

File: src/components/register/RegisterCardTwo.vue [hunk 1]
```diff
@@ -0,0 +1,149 @@
+<template>
+  <div class="register-card-wrapper">
+    <form @submit.prevent="submitForm" novalidate>
+      <div class="register-card-wrapper__form form-field">
+        <MainInput v-model="firstName" inputName="first-name" inputType="text"
+          >FIRST NAME</MainInput
```
Comment (szymonlicau) [inline src/components/register/RegisterCardTwo.vue]: By the way, a cool, small tip for you, especially if you would like to add such strings to i18n in the future 💡

We can actually not write text in a full caps like here, but normally `First name` and then make... CSS capitalize it for us 😄

```css
text-transform: uppercase;
```

If all your inputs have the labels in all caps - we can style it as such in the `MainInput` component itself. If not, maybe add a prop to sometimes capitalize it and apply a class to do so? 🚀

File: src/components/register/RegisterSteps.vue [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  <div>
+    <slot />
+  </div>
+</template>
+
+<script>
+export default {};
+</script>
+
+<style lang="scss" scoped></style>
```
Comment (szymonlicau) [inline src/components/register/RegisterSteps.vue:11]: Will this component be implemented later on to have more than just a `slot`? 🤔

File: src/pages/ScreeningsPage.vue [hunk 3]
```diff
@@ -57,7 +70,25 @@ export default {
...
+      return moviesWithGenre;
+    },
+    moviesByCategory() {
+      if (this.selectedGenre === "") {
+        return this.moviesOnTheDay;
+      } else {
+        return this.moviesOnTheDay.filter(
+          (movie) => movie.genre === this.selectedGenre
+        );
+      }
```
Comment (szymonlicau) [inline src/pages/ScreeningsPage.vue]: A small thing, but if you have a simple return in the `if () { }` condition, and a single `else` case then no need to wrap things with the `else { }` 💡 
```suggestion
      }
      return this.moviesOnTheDay.filter(
        (movie) => movie.genre === this.selectedGenre
      );
```

**Review by szymonlicau — CHANGES_REQUESTED** [review]

File: src/components/register/RegisterCardTwo.vue (hunk 1 above)
Comment (judytabaca) [reply src/components/register/RegisterCardTwo.vue]: this is a good tip!!! Thank you 💡

File: src/pages/ScreeningsPage.vue (hunk 3 above)
Comment (judytabaca) [reply src/pages/ScreeningsPage.vue]: thiis is very true 😨 thank you!!

File: src/components/register/RegisterCardTwo.vue [hunk 4]
```diff
@@ -0,0 +1,149 @@
+<template>
+  <div class="register-card-wrapper">
+    <form @submit.prevent="submitForm" novalidate>
+      <div class="register-card-wrapper__form form-field">
+        <MainInput v-model="firstName" inputName="first-name" inputType="text"
+          >Firts name</MainInput
```
Comment (szymonlicau) [inline src/components/register/RegisterCardTwo.vue:6]: Typo 🐷 
```suggestion
          >First name</MainInput
```

**Review by szymonlicau — CHANGES_REQUESTED** [review]

**Pull request closed without merging**